zerver/openapi/zulip.yaml

```diff
@@ -13680,6 +13756,41 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+    get:
+      operationId: get-user-group-members
+      summary: Get user group members
+      tags: ["users"]
+      description: |
+        Get the members of a [user group](/help/user-groups).
+
+        `GET {{ api_url }}/v1/user_groups/{user_group_id}/members`
+
+        **Changes**: New in Zulip 6.0 (feature level 127).
+      parameters:
+        - $ref: "#/components/parameters/UserGroupId"
+        - $ref: "#/components/parameters/DirectMemberOnly"
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - $ref: "#/components/schemas/SuccessDescription"
+                  - additionalProperties: false
+                    description: MANUALLY
```

Comment:
What does this do?  It doesn't appear in the end documentation.

Reply:
I copied this part from an existing endpoint, and it uses this. But I checked and it doesn't appear to be of any use. Should I modify the existing commits only to remove this?